great_tables/_data_color/base.py

```diff
@@ -456,6 +515,20 @@ def _color_name_to_hex(colors: List[str]) -> List[str]:
     return colors
 
 
+def _colorbrewer_palette_to_hex(palette: str) -> List[str]:
```

Comment:
It looks like these are always used after a `in` check. WDYT of us removing these, and using the underlying dictionaries directly?

e.g.

```python
COLORBREWER_NAME_TO_PALETTE[palette]
```

If you want, you could combine both dictionaries (`COLORBREWER_NAME_TO_PALETTE` and `VIRIDIS_NAME_TO_PALETTE`) into one big dictionary at the bottom of constants:

```python
ALL_PALETTES = {
    **COLORBREWER_NAME_TO_PALETTE,
    **VIRIDIS_NAME_TO_PALETTE,
}
```

Reply:
Yeah, the extra functions are really redundant especially since we have the checking beforehand. I’ll also combine the dicts into one. That’s a great idea.